packages/cf-component-select/src/Select.js

```diff
@@ -43,7 +105,7 @@ class Select extends React.Component {
     }
 
     return (
-      <div className={className}>
+      <div className={`cf-select ${className}`}>
```

Comment:
is it due interoperability you adde `cf-select`. I thought  we could migrate those styles into fela as well

Reply:
There are components in cf-www-next which assume the existence cf-select.

Reply:
I see 2 such instances, e.g. 
```
  '> .cf-select': {
    flex: 1,
    margin: '0 5px'
  },
```

 we should allow to configure this  through Theming.

```
import { SelectUnstyled, SelectTheme } from 'cf-component-select';

const CustomSelect = applyTheme(SelectUnstyled, SelectTheme, { margin: '0 5px',  flex: 1 });
```